### PR TITLE
PICARD-440: Tell the user they need to restart after changing plugins

### PR DESCRIPTION
**Pull request by yagyanshbhatia**
# Summary

* This is a…
    * [ ] Bug fix
    * [X] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [ ] Other

# Problem
* JIRA ticket (_optional_): [PICARD-440](https://tickets.metabrainz.org/browse/PICARD-440)

### CODE REVIEW

**Comment by samj1912** [comment]
@yagyanshbhatia please follow the PR template that comes on GH when you make the PR.

File: picard/ui/options/plugins.py [hunk 1]
```diff
@@ -280,7 +280,15 @@ def save(self):
         for item, plugin in self.items.items():
             if item.checkState(0) == QtCore.Qt.Checked:
                 enabled_plugins.append(plugin.module_name)
-        config.setting["enabled_plugins"] = enabled_plugins
+        if config.setting["enabled_plugins"] != enabled_plugins:
+            config.setting["enabled_plugins"] = enabled_plugins
+            dialog = QtWidgets.QMessageBox(
+                QtWidgets.QMessageBox.Information,
+                _('Plugins changed'),
+                _('You have changed the Plugins. You have to restart Picard in order for the change to take effect.'),
```
Comment (samj1912) [inline picard/ui/options/plugins.py]: I think imperative voice would be better here. Something like
> Enabled plugins have been changed. Please restart Picard in order for the change to take effect.

**Comment by yagyanshbhatia** [comment]
@samj1912 do you want me to change the commit message? or just the comment on PR. just one more thing, i can't find that template without creating another pull request :worried: . please tell me what needs to be done.

**Comment by samj1912** [comment]
Looking at the ticket comment, this seems unnecessary now.

**Comment by yagyanshbhatia** [comment]
Sorry I saw this now. Too bad it can't be merged :p

**Pull request closed without merging**